tests/test_selection/test_drop_high_psi_features.py

```diff
@@ -0,0 +1,410 @@
+from datetime import date
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.datasets import make_classification
+from sklearn.exceptions import NotFittedError
+
+from feature_engine.selection.drop_psi_features import DropHighPSIFeatures
+
+
+@pytest.fixture(scope="module")
+def df():
+    # create array with 4 correlated features and 2 independent ones
+    X, y = make_classification(
+        n_samples=1000,
+        n_features=6,
+        n_redundant=2,
+        n_clusters_per_class=1,
+        weights=[0.50],
+        class_sep=2,
+        random_state=1,
+    )
+
+    # transform array into pandas df
+    colnames = ["var_" + str(i) for i in range(6)]
+    X = pd.DataFrame(X, columns=colnames)
+
+    # Add drifted features that will be dropped during transformation.
+    X["drift_1"] = [itel for itel in range(X.shape[0])]
```

Comment:
what does itel stand for?

Reply:
That's a bad habit I have to call itel in an iteration. I changed it to number.